src/main/java/com/example/housingmanagement/api/controllers/HouseController.java

```diff
@@ -30,30 +30,23 @@ public ResponseEntity<List<HouseResponse>> getAllHouses() {
     public ResponseEntity<Void> addNewHouse(@RequestBody HouseRequest houseToBeAdded) {
         //checks if this house exists
         if (houseService.existsByHouse(houseToBeAdded)) {
-            System.out.println("A House with number " + houseToBeAdded.getHouseNumber() + " already exists");
             return ResponseEntity.badRequest().build();
         }
         houseService.addHouseToDatabase(houseMapper.toHouseInternalEntity(houseToBeAdded));
-        System.out.println("Added House with address: " + houseToBeAdded.getHouseNumber());
-
         return ResponseEntity.status(HttpStatus.CREATED).build();
     }
 
     @DeleteMapping(value = "/housing")
     public ResponseEntity<Void> deleteSpecificHouse(@RequestBody HouseRequest houseToBeDeleted) {
         //check if such house exists in Database
         if (!houseService.existsByHouse(houseToBeDeleted)) {
-            System.out.println("There is no house with number " + houseToBeDeleted.getHouseNumber() + " in the Database.");
-            return ResponseEntity.badRequest().build();
+           return ResponseEntity.badRequest().build();
```

Comment:
Tutaj się trzeba zasatnowić czy badRequest to dobra zwrotka? Raczej pasowałoby tutaj 422

Reply:
Dzięki - 422 czyli unprocessable entity